packages/protocol/migrations/16_elect_validators.ts

```diff
@@ -18,10 +23,29 @@ const argv = minimist(process.argv, {
   default: { keys: '' },
 })
 
+const gethRepoPath = argv.localgeth || '/tmp/geth'
```

Comment:
Is `/tmp/geth` a reasonable temporary geth path? Or should we require this argument and fail with a clear error message if not supplied?

Reply:
This is part of the code that I'm removing, I missed that line before the last commit.